cranelift/filetests/filetests/isa/x64/shift-i128-run.clif

```diff
@@ -71,3 +71,82 @@ block0(v0: i64, v1: i64, v2: i8):
 ; run: %rotr(0x01010101_01010101, 0x01010101_01010101, 73) == [0x80808080_80808080, 0x80808080_80808080]
 ; run: %rotr(0x01010101_01010101, 0x02020202_02020202, 0) == [0x01010101_01010101, 0x02020202_02020202]
 ; run: %rotr(0x01010101_01010101, 0x03030303_03030303, 128) == [0x01010101_01010101, 0x03030303_03030303]
+
+
+; i128 amount operand tests
+
+function %ishl_amt_i128(i64, i64, i8) -> i64, i64 {
+block0(v0: i64, v1: i64, v2: i8):
+    v3 = uextend.i128 v2
```

Comment:
Could we add some cases that test the masking as well? A simple way to do this might just be `v3 = uextend.i64 v2` / `v4 = iconcat v3, v3`, i.e. extend to 64 bits then replicate the 64 bits into the low and high parts.